client/dashboard/sites/backups/index.tsx

```diff
@@ -50,6 +59,40 @@ export function SiteBackupsCallout( {
 	);
 }
 
+function Backups( { site }: { site: Site } ) {
+	const [ view, setView ] = useState< View >( {
+		type: 'table',
+		fields: [ 'date', 'action', 'user' ],
+	} );
+
+	const { data: activityLog, isLoading: isLoadingActivityLog } = useQuery(
```

Comment:
Let's preload this query in the route. See: https://github.com/Automattic/wp-calypso/blob/4c1fe23fe698f57310e8de32482b3b5c87874219/client/dashboard/app/router.tsx#L97-L99